packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -128,7 +180,46 @@ describe('TransferDomain', () => {
       await expect(promise).rejects.toThrow('Src address must be a legacy or Bech32 address in case of "DVM" domain')
     })
 
-    it('(evm -> dvm) should fail if source address and source domain are not match', async () => {
+    it('(evm -> dvm) should fail if dst address is not legacy or Bech32 address in case of "DVM domain', async () => {
+      const promise = client.account.transferDomain([
+        {
+          src: {
+            address: evmAddr,
+            amount: '3@DFI',
+            domain: TransferDomainType.EVM
+          },
+          dst: {
+            address: evmAddr, // <- not match
+            amount: '3@DFI',
+            domain: TransferDomainType.DVM // <- not match
```

Comment:
need to recorrect to params here since this test is to test the dst addr type

Reply:
resolved